libs/goldbach.js

```diff
@@ -12,30 +12,74 @@ class goldbachFormulas {
   /**
   * @property {Function} set_max_number sets the number we're looking to find
   *           all additions for
-  * @param number the number we're going to process
+  * @param {number} number the number we're going to process
   */
-  set_max_number(number){
+  set_max_number(number) {
     this.__max_number = number;
   }
 
   /**
    * @property {Function} run_process executes the goldbach formulas in the
    *           class
+   * @return {function} goldbach formula
    */
-  run_process(){
-    console.log('Call other goldbach methods against ' + this.__max_number +
-        ' and output the result'
-    );
+  run_process() {
+    return this.__goldbach_formula();
   }
 
-  __get_primes(){
+  /**
+   * @property {Function} __get_primes executes the goldbach formulas in the
+   *           class
+   *  @param {int} n is prime reference
+   *  @return {int} primes
+   */
+  __get_primes(n) {
     // reference this.__max_number
+    const x = [];
+    let i = 0;
+    let j = 0;
+    const primes = [];
+    for (i = 2; i <= n; ++i) {
+      if (!x[i]) {
+        primes.push(i);
+        for (j = i << 1; j <= n; j += i) {
+          x[j] = true;
+        }
+      }
+    }
+    return primes;
   }
 
-  __goldbach_formula(){
+
+  /**
+* @property {Function} __goldbach_formula executes the goldbach formulas in the
+*           class
+*
+*@return {tupple} return tupples
+*/
+  __goldbach_formula() {
     // reference this.__max_number
-  }
 
+    const primes = this.__get_primes(this.__max_number);
+    //  console.log(primes)
+    const tupples = [];
+    const len = primes.length;
+    //  console.log(len);
+    for (let i = 0; i < len; i++) {
+      for (let j = len - 1; j >= i; j--) {
+        //  console.log(j)
+        // console.log(primes[i] + ": " + primes[j]);
+        if (primes[i] + primes[j] == this.__max_number) {
+          tupples.push(primes[i] + ' + ' + primes[j]);
```

Comment:
@MrBHerring 
Rather than pushing a string to the tupple, let's push an array of the two numbers.  It might make it easier for @infernoj to make unit tests against the function that way.
 ```suggestion
          tupples.push([primes[i],primes[j]]);
```